op-node/rollup/derive/l1_block_info.go

```diff
@@ -159,10 +164,24 @@ func (info *L1BlockInfo) unmarshalBinaryBedrock(data []byte) error {
 // | 32      | BlockHash                |
 // | 32      | BatcherHash              |
 // +---------+--------------------------+
-
+// Marshal Ecotone and Isthmus
 func (info *L1BlockInfo) marshalBinaryEcotone() ([]byte, error) {
-	w := bytes.NewBuffer(make([]byte, 0, L1InfoEcotoneLen))
-	if err := solabi.WriteSignature(w, L1InfoFuncEcotoneBytes4); err != nil {
+	out, err := marshalBinaryWithSignature(info, L1InfoFuncEcotoneBytes4)
+	if err != nil {
+		return nil, fmt.Errorf("failed to marshal Ecotone l1 block info: %w", err)
+	}
+	return out, nil
+}
+func (info *L1BlockInfo) marshalBinaryIsthmus() ([]byte, error) {
+	out, err := marshalBinaryWithSignature(info, L1InfoFuncIsthmusBytes4)
+	if err != nil {
+		return nil, fmt.Errorf("failed to marshal Isthmus l1 block info: %w", err)
+	}
+	return out, nil
+}
```

Comment:
nit: no newline

Reply:
@tynes not sure what you mean? do you want a new-line? the other set of 3 `unmarshal...` are all clumped together